pubsub/topic.go

```diff
@@ -100,6 +101,10 @@ type PublishSettings struct {
 	//
 	// Defaults to DefaultPublishSettings.BufferedByteLimit.
 	BufferedByteLimit int
+
+	// MessageRetentionDuration indicates the minimum duration to retain a message after
+	// it is published to the topic.
```

Comment:
Does this mean that the message may or may not be dropped after the `MinimumRetentionDuration` has passed? Is there any behavior that can be documented for end users to help them make an educated choice for this field?

Reply:
Messages are dropped after all of a topic's subscriptions have acknowledge the message. I'll add a link to the cloud docs.